### PR TITLE
Finishing mobile version of my portfolio

### PR DESCRIPTION
**Pull request by rivasbolinga**
Finishing the mobile version of my portfolio, adding section of projects and about me section

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -30,10 +30,134 @@ <h1 class="full-name">Andrea Rivas
...
         and have a project you need coded, don’t hesitate and contact me.
       </p>
-      <div class="start-collaboration">
+     
         <button class="btn-collaboration">Start collaboration</button>
-      </div>
+      
     </section>
     <div class="background-image"> </div>
+    <section class="works-container">
```
Comment (Felix45) [inline portfolio.html:38]: [ Required ] Please consider fixing the whitespace that appears on your design for screen sizes greater than `320px`, for example at a width of `375px` your implementation of the design introduces a white space on the right margin. Please see the screenshot below and rectify this issue.

## Figma template


## Current Implementation

File: portfolio.html [hunk 2]
```diff
@@ -30,10 +30,134 @@ <h1 class="full-name">Andrea Rivas
...
+            <li class="ruby-tag">Ruby on Rails</li>
+            <li class="js-tag">JavaScript</li>
+          </ul>
+         <button class="btn-see-project">See this project </button>
+        </div>
+       
+      </div>
+    </section>
+    <section class="about-me-section">
+      <div class="about-me-info">
```
Comment (Felix45) [inline portfolio.html:124]: [ Required ] Please consider fixing the whitespace that appears on your design for screen sizes greater than `320px`, for example at a width of `375px` your implementation of the design introduces a white space on the right margin. Please see the screenshot below and rectify this issue.

## Figma template


## Current implementation

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @rivasbolinga , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :heavy_check_mark: 
- [ ] All linter checks are passing :white_check_mark: 
- [ ] Descriptive pull request title and summary :white_check_mark: 
- [ ] Good descriptive commit messages :white_check_mark: 

 almost-there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fobadara — APPROVED** [review]
Hi @rivasbolinga, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**